2021-01-05 11:05:55,809 - [INFO] - __main__ - (app.py).(16) - [Bot] Бот начал работу!
2021-01-05 11:06:48,052 - [ERROR] - __main__ - (app.py).(140) - [ERROR] name 'user0' is not defined
2021-01-05 11:06:48,052 - [INFO] - __main__ - (app.py).(143) - [Bot] Бот закончил работу!
